### PR TITLE
feat: add relayerID and instance env to configuration to use in metrics

### PR DESCRIPTION
**Pull request by P1sar**
## Description

ENV and RELAYER_ID env params containing current instance environment and RelayerID correspondingly, these variables are needed to distinguish metric names from each other
## Related Issue Or Context

Closes: #<issue>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [x] I have updated the documentation locally and in docs.
- [x] I have added tests to cover my changes.
- [x] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

**Review by mpetrun5 — CHANGES_REQUESTED** [review]

File: metrics/metrics.go [hunk 1]
```diff
@@ -19,20 +20,20 @@ type Metrics struct {
...
 // NewMetrics creates an instance of metrics
-func NewMetrics(meter metric.Meter) *Metrics {
+func NewMetrics(meter metric.Meter, env, relayerID string) *Metrics {
 	totalRelayerCount := new(int64)
 	availableRelayerCount := new(int64)
 	return &Metrics{
 		ChainbridgeMetrics: *opentelemetry.NewChainbridgeMetrics(meter),
 		TotalRelayers: metric.Must(meter).NewInt64GaugeObserver(
-			"sygma.TotalRelayers",
+			fmt.Sprintf("sygma.{%s}.Relayer-{%s}.TotalRelayers", env, relayerID),
```
Comment (mpetrun5) [inline metrics/metrics.go:29]: Would it make more sense to have the id and env as context fields on the metric and not change metric name?
Also, do you need to add that to somemetrics from core?

Comment (P1sar) [reply metrics/metrics.go:29]: This sounds reasonable with context fields, although we are now not sure how to pass a meaningful context right now and just trying to make it work and build some alarm system with the OTLP agent and DataDog. 
About Core metrics, I am not sure we need the type of metrics served there, frankly speaking, those were pretty random. I do not want to use them for now

Comment (mpetrun5) [reply metrics/metrics.go:29]: We added meaningful stuff there in the mean time which we discussed we should have.

**Review by mpetrun5 — CHANGES_REQUESTED** [review]

**Review by mpetrun5 — APPROVED** [review]

**Review by MakMuftic — APPROVED** [review]

**Comment by P1sar** [comment]
Merging with Admin privileges bcs in recent PR we have updated the test requirement for `go 1.19` while here it is still `1.18`

**Pull request merged**